docker/scripts/hardware/utils.py

```diff
@@ -110,9 +110,15 @@ def get_smart_devices():
 
         dev, _, disk_type = line.split(" ")[0:3]
 
-        if "megaraid" not in disk_type:
+        if "megaraid" in disk_type:
```

Comment:
Hmm, I'm a little confused here...we're going from megaraid-only to anything-but-megaraid on this path?